### PR TITLE
docs: replace "Storybook" in title with "Castor"

### PR DESCRIPTION
**Pull request by rabelloo**
## Purpose

Have "Castor" in the browser title as a "suffix" rather than the default "Storybook"

## Approach

MutationObserver on `<title>` element

## Testing

Every story should have "Castor" as the browser title suffix

## Risks

None

### CODE REVIEW

File: .storybook/manager.ts [hunk 1]
```diff
@@ -4,8 +4,14 @@ export const mode = process.env.STORYBOOK_MODE; // defined on CI script
...
+// everytime Storybook changes the document title, replace it with "Castor"
+new MutationObserver(() => {
+  if (document.title.endsWith('Storybook'))
+    document.title = document.title.slice(0, -9) + 'Castor';
+}).observe(document.querySelector('title'), { childList: true });
+
 declare global {
   interface Window {
-    STORYBOOK_GA_ID: string;
+    STORYBOOK_GA_ID?: string;
```
Comment (josokinas) [inline .storybook/manager.ts:15]: And a little unrelated change 🙈

**Review by josokinas — APPROVED** [review]

File: .storybook/manager.ts (hunk 1 above)
Comment (rabelloo) [reply .storybook/manager.ts:15]: Whoops, staging parts of the same file in different commits gets me every time.
Will revert this line

Comment (rabelloo) [inline .storybook/manager.ts:15]:
```suggestion
    STORYBOOK_GA_ID: string;
```
